Recurly/Resources/Account.cs

```diff
@@ -17,94 +20,117 @@ public class Account : Resource
+
         [JsonProperty("bill_to")]
-        public string BillTo { get; set; }
+        [JsonConverter(typeof(RecurlyStringEnumConverter))]
+        public BillToEnum? BillTo { get; set; }
 
 
         [JsonProperty("billing_info")]
+
         public BillingInfo BillingInfo { get; set; }
 
         /// <value>Additional email address that should receive account correspondence. These should be separated only by commas. These CC emails will receive all emails that the `email` field also receives.</value>
         [JsonProperty("cc_emails")]
+
         public string CcEmails { get; set; }
 
         /// <value>The unique identifier of the account. This cannot be changed once the account is created.</value>
         [JsonProperty("code")]
+
         public string Code { get; set; }
 
 
         [JsonProperty("company")]
+
         public string Company { get; set; }
 
         /// <value>When the account was created.</value>
         [JsonProperty("created_at")]
+
         public DateTime? CreatedAt { get; set; }
 
 
         [JsonProperty("custom_fields")]
+
         public List<CustomField> CustomFields { get; set; }
 
         /// <value>If present, when the account was last marked inactive.</value>
         [JsonProperty("deleted_at")]
+
         public DateTime? DeletedAt { get; set; }
 
         /// <value>The email address used for communicating with this customer. The customer will also use this email address to log into your hosted account management pages. This value does not need to be unique.</value>
         [JsonProperty("email")]
+
         public string Email { get; set; }
 
         /// <value>The tax exemption certificate number for the account. If the merchant has an integration for the Vertex tax provider, this optional value will be sent in any tax calculation requests for the account.</value>
         [JsonProperty("exemption_certificate")]
+
         public string ExemptionCertificate { get; set; }
 
 
         [JsonProperty("first_name")]
+
         public string FirstName { get; set; }
 
         /// <value>The unique token for automatically logging the account in to the hosted management pages. You may automatically log the user into their hosted management pages by directing the user to: `https://{subdomain}.recurly.com/account/{hosted_login_token}`.</value>
         [JsonProperty("hosted_login_token")]
+
         public string HostedLoginToken { get; set; }
 
 
         [JsonProperty("id")]
+
         public string Id { get; set; }
 
 
         [JsonProperty("last_name")]
+
         public string LastName { get; set; }
 
         /// <value>The UUID of the parent account associated with this account.</value>
         [JsonProperty("parent_account_id")]
+
         public string ParentAccountId { get; set; }
 
-        /// <value>Used to determine the language and locale of emails sent on behalf of the merchant to the customer.</value>
```

Comment:
We need to preserve these docstrings if we can

Reply:
Fixed and apparently pulled in some new ones as well!